exercism_test_helper/lib/meta/style.ex

```diff
@@ -0,0 +1,62 @@
+defmodule Meta.Style do
+  @function_without_parens [
+    :assert,
+    :assert_in_delta,
+    :assert_raise,
+    :assert_receive,
+    :assert_received,
+    :catch_error,
+    :catch_exit,
+    :catch_throw,
+    :flunk,
+    :refute,
+    :refute_in_delta,
+    :refute_receive,
+    :refute_received
+  ]
+
+  def format(ast), do: Macro.to_string(ast, &format/2)
+
+  # Custom formatting handler for `if` and `unless` expressions
```

Comment:
Do `cond` and `case` also need special treatment here?

Reply:
surprisingly no, iirc, but maybe I should add a test to verify